client/src/components/Post/Post.tsx

```diff
@@ -266,11 +294,27 @@ class Post extends Component<Props, State> {
                     </div>
                     <div className={'body-footer'}>
                         <div className={'like-container'}>
-                            <MdThumbUp size={20} color={'#00B090'} className={'like-icon'}/>
+                            { this.state.clickedUpvote&&
+                                <MdThumbUp size={20} color={'#00B090'} className={'like-icon'}
+                                           onClick={() => this.handleUpvote()}/>
+                            }
+                            {!this.state.clickedUpvote&&
+                                <MdThumbUp size={20} color={'grey'} className={'like-icon'}
+                                           onClick={() => this.handleUpvote()}/>
+                            }
```

Comment:
We could use ternary operator for this ex: { this.state.clickedUpvote? <> ...etc</> : <> ...etc</>}